tajo-common/src/main/java/org/apache/tajo/type/Type.java

```diff
@@ -36,6 +36,7 @@
 
   // No paramter types
   public static final Any Any = new Any();
+  public static final Bit Bit = new Bit();
```

Comment:
How about making the modifier of the constructors of these singleton instances to be ```default```?
This is not the changes made in this patch, but it seems better.